Jak-Sidious/RestPlusApi
# Crud functionality 2

#### What does this PR do?

Merge the existing branch into develop as the current functionality stands at 74%

#### Description of Task to be completed?

Implemented basic crud Functionality as well as basic testing 

#### How should this be manually tested?

after cloning the repo locally, and installing the requirements the following code can be run to check on the current coverage

`nose2 --with-cov --coverage app`
